Main.java

```diff
@@ -0,0 +1,18 @@
+public class Main {

```

Comment:
What are the important aspects of writing code?
What is tough in programming?
Concurrency? Internals?